packages/loot-core/src/types/api-handlers.d.ts

```diff
@@ -143,4 +143,14 @@ export interface ApiHandlers {
   'api/payee-update': (arg: { id; fields }) => Promise<unknown>;
 
   'api/payee-delete': (arg: { id }) => Promise<unknown>;
+
+  'api/rules-get': () => Promise<RuleEntity[]>;
+
+  'api/payee-rules-get': (arg: { id }) => Promise<RuleEntity[]>;
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏missing argument type (here and also the other API methods below)
```suggestion
  'api/payee-rules-get': (arg: { id: string }) => Promise<RuleEntity[]>;
```